src/models/wikiPageListing.es6.js

```diff
@@ -0,0 +1,23 @@
+import Base from './base';
+
+class WikiPageListing extends Base {
+  constructor(props) {
+    props.pages = props.data.slice();
```

Comment:
you should probably also `delete props.data` so you don't end up with two copies